test/controllers/IllnessStartDateController.test.ts

```diff
@@ -0,0 +1,156 @@
+import { expect } from 'chai';
+import { MOVED_TEMPORARILY, OK, UNPROCESSABLE_ENTITY } from 'http-status-codes';
+import { before } from 'mocha';
+import request from 'supertest';
+
+import 'app/controllers/EvidenceDownloadController';
+import {
+    ENTRY_PAGE_URI,
+    ILLNESS_START_DATE_PAGE_URI
+} from 'app/utils/Paths';
+
+import { createApp } from 'test/ApplicationFactory';
+
+const pageHeading: string = 'When did the illness start?';
+const errorSummaryHeading: string = 'There is a problem';
+const invalidStartDayErrorMessage: string = 'You must enter a day';
+const invalidStartMonthErrorMessage: string = 'You must enter a month';
+const invalidStartYearErrorMessage: string = 'You must enter a year';
+const invalidDateErrorMessage: string = 'Enter a real date';
+const invalidDateFutureErrorMessage: string = 'Start date must be today or in the past';
+
+describe('IllnessStartDateController', () => {
+
+    describe('GET request', () => {
+
+        before(() => {
+            process.env.ILLNESS_REASON_FEATURE = '1';
+        });
```

Comment:
`before` in this case only is applied to two tests, but one of them overides it by seting the feature to 0. So this is only really useful for one test, id move this inside that test